docs/imagecustomizer/how-to/quick-start.md

```diff
@@ -0,0 +1,70 @@
+---
+parent: How To
+nav_order: 4
+---
+
+# Quick start
+
+1. Download an Azure Linux core VHDX image file.
+
+2. Create a customization config file.
+
+   For example:
+
+    ```yaml
+    os:
+      packagesInstall:
+      - dnf
+    ```
+
+   For documentation on the supported configuration options, see:
+   [Supported configuration](./api/configuration.md)
+
+3. Install prerequisites: `qemu-img`, `rpm`, `dd`, `lsblk`, `losetup`, `sfdisk`,
+   `udevadm`, `flock`, `blkid`, `openssl`, `sed`, `createrepo`, `mksquashfs`,
+   `genisoimage`, `parted`, `mkfs`, `mkfs.ext4`, `mkfs.vfat`, `mkfs.xfs`, `fsck`,
+   `e2fsck`, `xfs_repair`, `resize2fs`, `tune2fs`, `xfs_admin`, `fatlabel`, `zstd`,
+   `veritysetup`, `grub2-install` (or `grub-install`).
+
+   - For Ubuntu 22.04 images, run:
+
+     ```bash
+     sudo apt -y install qemu-utils rpm coreutils util-linux mount fdisk udev openssl \
+        sed createrepo-c squashfs-tools genisoimage parted e2fsprogs dosfstools \
+        xfsprogs zstd cryptsetup-bin grub2-common
+     ```
+
+   - For Mariner 2.0, run:
```

Comment:
update to `Azure Linux 2.0`

Reply:
Should we update to `3.0` actually?

Reply:
The list works for both. So, added both to doc.